### ISSUE TITLE
output-file should be determined from config-file if present

### ISSUE THREAD

**Issue by flatiron32** [terraform-docs/gh-actions#62]
### What problem are you facing?

I would like to be able to keep all of the general terraform-docs configuration in the config-file, .terraform-docs.yml for instance. When I have an output file defined in .terraform-docs.yml and that set as the config-file for the action, the output file is updated as expected but the default USAGE.md is created and that injected content is `README.md updated successfully`. The README.md is not added to the change set and pushed when `git-push` is configured.

The config file should be parsed by the action to determine if the output file is defined there. If so, use that.

With `.terraform-docs.yml` of
```yml
formatter: markdown table

output:
  file: README.md
```
I want to configure the action step as
```yml
      - name: Generating Documentation
        uses: terraform-docs/gh-actions@v0.10.0
        with:
          config-file: .terraform-docs.yml
          git-push: "true"
```
I expect README.md to be updated and pushed when the action runs.

**Comment by khos2ow** [comment]
This is an issue with gh-actions that does not (yet) support native output to file from terraform-docs. It will be fixed by #46.

Also keep an eye on the release note, as this will be a breaking change (unless I find a way to make it backward compatible)

**Comment by khos2ow** [comment]
This is now fixed as part of #46 and will be available in the upcoming `v1.0.0` release.

**Comment by jverce** [comment]
From [the `terraform-docs` docs](https://terraform-docs.io/user-guide/configuration/):
> Values passed directly as CLI flags will override all of the above.

Given that the CLI args will basically override whatever's in the config file, should we avoid default values for those args altogether?

Otherwise, we need to explicitly pass the proper value for every CLI argument, even if they are already defined in the config file, thus making such file pointless.
